app/views/users/new.html.erb

```diff
@@ -0,0 +1,47 @@
+
+<%# <form action="/users" method="post" accept-charset="UTF-8">
+<input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>
+  <%# <label for="username">Username:</label>
+  <input type="text" id="username" name="username"><br><br>
+  <label for="email">Email:</label>
+  <input type="email" id="email" name="email"><br><br>
+  <label for="password">Pasword:</label>
+  <input type="password" id="password" name="password"><br><br>
+  <input type="submit" value="Submit">
+</form> %>
+
+<%# <%= form_tag("/users", method: "post") do %> 
```

Comment:
- [ ] Use `users_path` for the form_tag only parameter. Adding the method won't be needed then.